hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcServerResponseEncoder.java

```diff
@@ -31,6 +31,8 @@
 @InterfaceAudience.Private
 class NettyRpcServerResponseEncoder extends ChannelOutboundHandlerAdapter {
 
+  static final String HANDLER_NAME = "NettyRpcServerResponseEncoder";
```

Comment:
nit: rename it as "ENCODER_NAME"?

Reply:
@wchevreuil, thanks for review，here I did not name it as "ENCODER_NAME" because it is misleading  and is also inconsistent with the naming conventions of other netty handlers,  here this name represents `NettyRpcServerResponseEncoder` itself，but for other netty handler such as ``NettyRpcServerPreambleHandle`,`NettyRpcServerPreambleHandler.DECODER_NAME` represents decoder handler before it, not represents itself.  I have ready renamed it as "NAME" to keep consistent with the naming conventions of other netty handlers.